client/src/pages/spools/show.tsx

```diff
@@ -58,6 +58,8 @@ export const SpoolShow: React.FC<IResourceComponentsProps> = () => {
       <NumberField value={record?.id ?? ""} />
       <Title level={5}>{t("spool.fields.filament")}</Title>
       <TextField value={record ? filamentURL(record?.filament) : ""} />
+      <Title level={5}>{t("spool.fields.price")}</Title>
+      <NumberField value={record?.price ?? ""} />
```

Comment:
Here you should also default to the filament's price, to make it consistent with the list view

Reply:
Filament price is now shown on spools/show.tsx when spool price is not set :)